testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1047,7 +1047,7 @@
     end
   # debian based systems don't come with curl installed
   if (os_family.include? 'debian') || (os_family.include? 'ubuntu')
-    node.run_until_ok("wget --no-check-certificate -qO- #{protocol}://#{system_name}:#{port}#{path} | grep -i '#{text}'")
+    node.run_until_ok("curl --insecure -s #{protocol}://#{system_name}:#{port}#{path} | grep -i '#{text}'")
   else
     node.run_until_ok("curl -s -k #{protocol}://#{system_name}:#{port}#{path} | grep -i '#{text}'")
```

Comment:
can't we just remove the if condition in here and use what's in the else for this and the other cases?

Reply:
wait, but actually, if debian based systems don't come with wget installed, can we even run this there?

Reply:
Yup, Vanda is right, we will need to install `curl` in debian-like minions, probably through cloud-init phase.

Reply:
Sorry, I did not realize there was an if ... the issue is that wget is not in Leap Micro, so we do not really need this change here.

Reply:
fixed in https://github.com/uyuni-project/uyuni/pull/8391